requirements-base.txt

```diff
@@ -49,7 +49,7 @@ requests[security]>=2.18.4,<2.19.0
 selenium==3.11.0
 semaphore>=0.1.0,<0.2.0
 simplejson>=3.2.0,<3.9.0
-six>=1.10.0,<1.11.0
```

Comment:
Didn't you wanna roll this back?

Reply:
I just opened this PR before I was done testing in https://github.com/getsentry/getsentry/pull/1947. It didn't turn out, so yeah.